File: L010/2dArrayDemo.java
import java.io.*;
import java.util.*;
import java.util.Scanner;

public class 2dArrayDemo{
public static Scanner sc = new Scanner(System.in);
public static void main(String[] args) throws Exception {
    // write your code here
    int n = sc.nextInt();
    int m = sc.nextInt();
    
    int[][] arr = new int[n][m];
    for(int i=0;i<arr.length;i++){
        for(int j=0;j<arr[0].length;j++){
            arr[i][j] = sc.nextInt();
        }
    }
    
    for(int i=0;i<arr.length;i++){
        for(int j=0;j<arr[0].length;j++){
            System.out.print(arr[i][j] + " ");
        }
        System.out.println();
    }
    
 }

}

File: L010/2dArrayDemo.cpp
#include<iostream>
#include<vector>
using namespace std;

int main(){
    int n,m;
    cin>>n>>m;
    
    vector<vector<int>>arr(n,vector<int>(m,0));
    for(int i=0;i<arr.size();i++){
        for(int j=0;j<arr[0].size();j++){
            cin>>arr[i][j];
        }
    }
    
    
    for(int i=0;i<arr.size();i++){
        for(int j=0;j<arr[0].size();j++){
            cout<<arr[i][j]<<" ";
        }
        cout<<endl;
    }
    
    return 0;
}